code/ui/.storybook/preview.jsx

```diff
@@ -0,0 +1,248 @@
+import global from 'global';
+import React, { Fragment, useEffect } from 'react';
+import isChromatic from 'chromatic/isChromatic';
+import {
+  Global,
+  ThemeProvider,
+  themes,
+  createReset,
+  convert,
+  styled,
+  useTheme,
+} from '@storybook/theming';
+import { Symbols } from '@storybook/components';
+
```

Comment:
This file is mostly just copied from the old official storybook example. I did however remove the logic for showing warnings about missing env.